samples/highcharts/demo/music-race-chart/demo.js

```diff
@@ -0,0 +1,196 @@
+const btn = document.getElementById('play-pause-button'),
+    input = document.getElementById('play-range'),
+    startYear = 1973,
+    endYear = 2021;
+
+// General helper functions
+const getRevenue = (record, year) => {
+    const revenue = record[year];
+    if (typeof revenue !== 'undefined') {
+        return revenue;
+    }
+    return 0;
+};
+
+const arrToAssociative = arr => {
+    const tmp = {};
+    arr.forEach(item => {
+        if (item[1] !== null) {
+            tmp[item[0]] = item[1];
+        }
+    });
+
+    return tmp;
+};
+
+const formatRevenue = [];
+
+const chart = Highcharts.chart('container', {
+    chart: {
+        type: 'area'
+    },
+    title: {
+        text: 'Music revenue race chart'
+    },
+    data: {
+        csv: document.getElementById('csv').innerHTML,
+        itemDelimiter: '\t',
+        complete: function (options) {
+            for (let i = 0; i < options.series.length; i++) {
+                formatRevenue[i] = arrToAssociative(options.series[i].data);
+                options.series[i].data = null;
+            }
+        }
+    },
+    xAxis: {
+        allowDecimals: false,
+        min: startYear,
+        max: endYear
+    },
+    yAxis: {
+        title: {
+            text: 'Revenue in the U.S.'
+        },
+        labels: {
+            format: '${text} B'
+        }
+    },
+    tooltip: {
+        pointFormat: '{series.name} had a revenue of <b>${point.y:,.1f} B</b><br/> in {point.x}'
+    },
+    plotOptions: {
+        area: {
+            stacking: 'normal',
+            pointStart: startYear,
+            pointEnd: endYear,
+            marker: {
+                enabled: false
+            },
+
+            label: {
+                style: {
+                    transition: 'transform 0.4s, opacity 0.8s'
+                }
+            }
+        }
+    },
+
+    series: [{
+        label: {
+            style: {
+                fontSize: '0.8em'
+            }
+        }
+    }, {
+        label: {
+            style: {
+                fontSize: '0.3em'
+            }
+        }
+    }, {
+        label: {
+            style: {
+                fontSize: '0.4em'
+            }
+        }
+    }, {
+        label: {
+            style: {
+                fontSize: '2em'
+            }
+        }
+    }, {
+        label: {
+            style: {
+                fontSize: '0.5em'
+            }
+        }
+    }, {
+        label: {
+            style: {
+                fontSize: '0.5em'
+            }
+        }
+    }]
+
+});
+
+function pause(button) {
+    button.title = 'play';
+    button.className = 'fa fa-play';
+    clearTimeout(chart.sequenceTimer);
+    chart.sequenceTimer = undefined;
+}
+
+function update() {
+    chart.update({
+        subtitle: {
+            text: 'Year ' + input.value
```

Comment:
The year kind of disappears. Can we put it in a big fat font somewhere on top of the chart, like we did with the [bar race](https://www.highcharts.com/demo/bar-race/brand-dark)?